pyproject.toml

```diff
@@ -9,6 +9,9 @@ python = "^3.7"
 flask = "^1.1.2"
 python-dotenv = "^0.14.0"
 requests = "^2.24.0"
+pytest = "^6.2.1"
+selenium = "^3.3.0"
+pytest-mock = "^3.5.1"
```

Comment:
You should move these to the dev-dependencies section

(Could have added them with `--dev` when doing `poetry add`) 

Also looks like `pytest-mock` is unused so should be removed. You are only using the built-in `unittest.mock` rather than pytest-mock's `mocker`